test/approximate-size-test.js

```diff
@@ -116,7 +116,3 @@ test('test approximateSize()', function (t) {
     })
   })
 })
-
-test('tearDown', function (t) {
-  db.close(testCommon.tearDown.bind(null, t))
```

Comment:
`testCommon.tearDown` should still be called though, right? To cleanup directories.

Reply:
Indeed. Just noticed it will create the db under `node_modules/`

Reply:
Removed the other `db.close()` instead :wink: